plugin/src/main/scala/org/scalafmt/sbt/Difference.scala

```diff
@@ -0,0 +1,71 @@
+package org.scalafmt.sbt
+
+import java.io.File
+
+import sbt.io.IO
+import sbt.util.{CacheStore, ChangeReport, FileInfo, FilesInfo, Tracked}
+
+// Originally copied from https://github.com/sbt/util/blob/7431dbd/util-tracking/src/main/scala/sbt/util/Tracked.scala
```

Comment:
I made a separate commit to import the file as-is for better diff tracking, but from what I can see the PR commits are squashed together when merging to master - would it be possible to keep history to better see what has changed?

Reply:
There is no official politic about squashing, but I prefer one commit per feature in master. If there is a need, I can merge it as is.

Reply:
Let's see whether we go for 9afe28b or the adhoc `sbt.util.Differences` (as explained in the description). If we go for the latter, I think it makes sense to keep a separate commit for the inlining of the sbt helper as-is (and squash 9afe28b & b3505b5 since 9afe28b is just an attempt not chosen).

Reply:
Can we reuse sbt-util and remove this adhoc after https://github.com/sbt/util/pull/205 merging? If yes, please add link to PR in comment. 

I subscribed to PR updates and try to refactor this as soon as sbt-util will be released, but link may still be useful.